### PR TITLE
add expansion module

### PR DESCRIPTION
**Pull request by fouille**
add new expansion keys for new devices

### CODE REVIEW

**Comment by fouille** [comment]
tested

File: plugins/wazo-yealink/v86/common.py [hunk 1]
```diff
@@ -274,6 +278,8 @@ def _nb_linekey_by_model(self, model):
     def _expmod_by_model(self, model):
         if model in ('T27G', 'T46S', 'T48S'):
             return self.EXP40ExpansionModule
+        elif model in (u'T43U', u'T46U', u'T48U'):
```
Comment (bloom1) [inline plugins/wazo-yealink/v86/common.py:281]: We are now in Python3. No more `u`! :smile:

**Review by bloom1 — CHANGES_REQUESTED** [review]

File: plugins/wazo-yealink/v86/common.py (hunk 1 above)
Comment (fouille) [reply plugins/wazo-yealink/v86/common.py]: @bloom1 thank's, fixed.

**Review by bloom1 — APPROVED** [review]
Thank you @fouille :smiley:

**Pull request merged**